domains/wendy.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "adduserwyw",
+    "email": "wendy@wendyyw.dev"
+  },
+  "records": {
+    "CNAME": "https://www.wendyyw.dev/"
```

CNAME doesnt allow any trailing slashes
```suggestion
    "CNAME": "www.wendyyw.dev"
```